### ISSUE TITLE
Update OpenCV library version

### ISSUE THREAD

**Issue by pnutmath** [andreluisjunqueira/react-native-documentscanner-android#33]
There are couple of years and 8516 commits after release of 3.1.0, we can update OpenCV library to latest version.

**Comment by andreluisjunqueira** [comment]
No, if you update the version problably the library will stop working. To update the library you have to update the classes within the library. Pull request are welcome !
